app/webpacker/components/Panel/pages/RunValidatorsPage/ValidationListView.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useMemo } from 'react';
+import _ from 'lodash';
+import { Header, List, ListItem } from 'semantic-ui-react';
+import I18n from '../../../../lib/i18n';
+import { competitionUrl } from '../../../../lib/requests/routes.js.erb';
+
+const headingPrefixForType = (type) => {
+  switch (type) {
+    case 'info': return 'Information for';
+    case 'warning': return 'Warnings detected in';
+    case 'error': return 'Errors detected in';
+    default: return 'Unknown detected in';
+  }
+};
+
+export default function ValidationListView({ validations, showCompetitionNameOnOutput, type }) {
+  const listByGroup = useMemo(() => _.groupBy(validations, 'kind'), [validations]);
+
+  return (
+    <>
+      {Object.entries(listByGroup).map(([group, list]) => (
+        <>
+          <Header as="h5">{`${headingPrefixForType(type)} ${group}`}</Header>
+          <List bulleted>
+            {list.map((validationData) => (
+              <ListItem>
+                <ValidationText
+                  validationData={validationData}
+                  group={group}
+                  showCompetitionNameOnOutput={showCompetitionNameOnOutput}
+                />
+              </ListItem>
+            ))}
+          </List>
+        </>
+      ))}
+    </>
+  );
+}
+
+function ValidationText({ validationData, group, showCompetitionNameOnOutput }) {
+  return (
+    <>
+      {showCompetitionNameOnOutput && (
+        <>
+          [
+          <a href={competitionUrl(validationData.competition_id)}>
+            {validationData.competition_id}
+          </a>
+          {'] '}
```

Comment:
Why do you need the `[]`? Are they absolutely essential for user communication?

Reply:
Huh, interesting that the `[` above is acceptable but `]` is not. Are there HTML codes (similar to `&apost;`) that you could use?